index.bs

```diff
@@ -318,27 +315,29 @@ Abstract Operations {#abstract-operations}
 
 Automation {#automation}
 ==========
-This section extends the [=automation=] section defined in the Generic Sensor API [[GENERIC-SENSOR]]
-to provide mocking information about the [=geolocation=] of the hosting device for the purposes of
-testing a user agent's implementation of {{GeolocationSensor}} API.
 
-<h3 id="mock-geolocation-sensor-type">Mock Sensor Type</h3>
+This section extends [[GENERIC-SENSOR#automation]] by providing [=Geolocation Sensor=]-specific virtual sensor metadata.
 
-The {{GeolocationSensor}} class has an associated [=mock sensor type=] which is
-<a for="MockSensorType" enum-value>"geolocation"</a>, its [=mock sensor reading values=]
-dictionary is defined as follows:
+The [=per-type virtual sensor metadata=] [=map=] must have the following [=map/entry=]:
+: [=map/key=]
+:: "`geolocation`"
+: [=map/value=]
+:: A [=virtual sensor metadata=] whose [=virtual sensor metadata/virtual sensor type=] is [=Geolocation Sensor=] and [=reading parsing algorithm=] is [=geolocation reading parsing algorithm=].
```

Comment:
```suggestion
:: A [=virtual sensor metadata=] whose [=virtual sensor metadata/virtual sensor type=] is [=Geolocation Sensor=] and [=reading parsing algorithm=] is the [=geolocation reading parsing algorithm=].
```

Reply:
Fixed in https://github.com/w3c/geolocation-sensor/pull/56/commits/c80e90090f305f6b5d701c9cee73648487ac8181